entrypoint.py

```diff
@@ -1,7 +1,23 @@
 import datetime
 import re
-from mescobrad_edge.plugins.questionnaire_anonymisation_plugin.models.plugin import EmptyPlugin, PluginActionResponse, PluginExchangeMetadata
+from mescobrad_edge.plugins.questionnaire_anonymisation_plugin.models.plugin import EmptyPlugin, PluginActionResponse, \
+    PluginExchangeMetadata
 from datetime import date
+from enum import Enum
+
+
+class DataTypes(Enum):
```

Comment:
Also, in the UI there is option NUMBER, so it needs to be checked to which option(number) this is mapped. And also to add check for this.